### PR TITLE
Minor change in People and AddPeople pages

### PR DESCRIPTION
**Pull request by BAAHUBALI7781**

### CODE REVIEW

File: src/views/AddPeople.vue [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <div class="addblog">
     <div class="container">
       <div class="card card-container">
-        <h1>Enter a new Entry</h1>
-        <h1>{{ this.id }}</h1>
+        <h1 id="header">Enter a new Entry</h1>
+        <h1 id="header">{{ this.id }}</h1>
```
Comment (shivam-51) [inline src/views/AddPeople.vue]: Can you remove this please?
This was added as a part of experiment, I forgot to remove later ;-)

**Review by shivam-51 — APPROVED** [review]
+1.
Check my inline comment.

File: src/views/AddPeople.vue (hunk 1 above)
Comment (shivam-51) [reply src/views/AddPeople.vue]: Remove the ` <h1 id="header">{{ this.id }}</h1>`

Comment (BAAHUBALI7781) [reply src/views/AddPeople.vue:6]: If you are talking about the "header" id, then they are added by me. I wanted to align the headers at the centers so did that.

Comment (shivam-51) [reply src/views/AddPeople.vue]: > Remove the ` <h1 id="header">{{ this.id }}</h1>`

@BAAHUBALI7781  What's the confusion? I wrote it quite clear to remove the above line :confused:

**Comment by BAAHUBALI7781** [comment]
Done @shivam-51

**Review by shivam-51 — APPROVED** [review]

**Pull request merged**